themes/gatsby-theme-blog/src/components/layout.js

```diff
@@ -1,72 +1,130 @@
-/* eslint-disable */
-import React from 'react'
-import { Link } from 'gatsby'
+import React from "react";
+import { Link } from "gatsby";
+import Toggle from "./toggle";
+// import Helmet from "react-helmet";
+import ThemeContext from "../context/theme-context";
 
-import { rhythm, scale } from '../utils/typography'
+import { rhythm, scale } from "../utils/typography";
+import sun from "../../content/assets/sun.png";
+import moon from "../../content/assets/moon.png";
 
-class Template extends React.Component {
-  render() {
-    const { location, children } = this.props
-    const rootPath = `${__PATH_PREFIX__}/`
-    let header
+const renderHeader = props => {
```

Comment:
Bikeshed: should we create 2 components instead?
Feel free to ignore but I think this is more readable

```js
const HomepageHeader = (props) => (
  <h1
    style={{
      ...scale(0.75),
      marginBottom: 0,
      marginTop: 0
    }}
  >
    <Link
      style={{
        boxShadow: "none",
        textDecoration: "none",
        color: "var(--textTitle)"
      }}
      to={"/"}
    >
      {props.title}
    </Link>
  </h1>
);
```

```js
const Header = (props) => (
  <h3
    style={{
      fontFamily: "Montserrat, sans-serif",
      marginTop: 0,
      marginBottom: 0,
      height: 42, // because
      lineHeight: "2.625rem"
    }}
  >
    <Link
      style={{
        boxShadow: "none",
        textDecoration: "none",
        color: "rgb(102, 185, 191)"
      }}
      to={"/"}
    >
      {props.title}
    </Link>
  </h3>
);
```

```js
return location.pathname === rootPath ? <HomePageHeader {...props} /> : <Header {...props} />
```

Reply:
(i have seen this, @wardpeet! just leaving it alone while we're figuring some patterns out 😸)